tests/test_cursor_server_async.py

```diff
@@ -572,7 +572,7 @@ async def test_stolen_cursor_close(aconn):
 
 
 async def test_row_maker_returns_none(aconn):
-    cur = aconn.cursor(row_factory=rows.scalar_row)
+    cur = aconn.cursor("test", row_factory=rows.scalar_row)
```

Comment:
Ah I missed this. It is something to review outside this MRm

Reply:
On it.

Reply:
Addressing it in 00e67ff4. Only the test was wrong, the library behaviour is correct.